configuration.c

```diff
@@ -1485,6 +1486,7 @@ static struct config_uint_setting *populate_settings_uint(settings_t *settings,
 #endif
    SETTING_UINT("audio_out_rate",               &settings->uints.audio_out_rate, true, out_rate, false);
    SETTING_UINT("custom_viewport_width",        &settings->video_viewport_custom.width, false, 0 /* TODO */, false);
+   SETTING_UINT("CRT_Switch_Resolution_super",  &settings->uints.CRT_Switch_Resolution_super,      true, CRT_Switch_Resolution_super, false);//ben
```

Comment:
please don't use uppercase for variables, it will trigger every dev's OCD

Reply:
Also @alphanu1 no C99 comments are allowed in the codebase as we still have to support C89-only platforms.

Reply:
No worries i'll run though and change things.